docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -233,6 +233,7 @@ metadata:
 spec:
   reference:
     export:
+      path: "root:compute"
```

Comment:
Is  that related to the si/non replacement ?

Reply:
No, it is fixing a distinct bug.